api/src/index.js

```diff
@@ -29,3 +29,70 @@ app.use("/api", apiRouter);
 app.listen(process.env.PORT, () => {
   console.log(`API listening on port ${process.env.PORT}`);
 });
+
+app.get("/my-route", (req, res) => {
+  res.send("Hi friend");
+});
+
+apiRouter.get("/future-meals", async (req, res) => {
+  try{
+    const GET_FUTURE_MEALS_QUERY = "SELECT * FROM meal WHERE `when` > NOW();";
+    const [futureMeals] = await knex.raw(GET_FUTURE_MEALS_QUERY);
```

Comment:
```suggestion
    const [futureMeals, schema] = await knex.raw(GET_FUTURE_MEALS_QUERY);
```
The `knex.raw()` call returns the rows as an array as the first return value, and then the schema as the second one. You should only return the result, not the schema.